src/views/DefineCompany.vue

```diff
@@ -39,6 +39,20 @@
         @contactInfoChange="onBusinessContactInfoChange($event)"
         @contactInfoFormValidityChange="onBusinessContactFormValidityChange($event)"/>
     </section>
+    <section class="mt-10" v-if="isEntityType && isPremiumAccount">
+      <header id="folio-number-header">
+        <h2>4. Folio / Reference Number (optional)</h2>
+        <p>Add an optional Folio or Reference Number about this business for your own tracking purposes.
+           This information is not used by the BC Business Registry.
+        </p>
+      </header>
+      <v-card flat class="step-container">
+          <FolioNumber
+            :initialValue="folioNumber"
+            @folioNumberChange="onFolioNumberChange($event)"
+          />
+      </v-card>
+    </section>
   </div>
 </template>
 
```

Comment:
Is there a read-only (or isEditing=false) version of this (for review page 5)?

Reply:
There is some text in the mock up mentioning this for step 5 that I missed. I will add this.

Reply:
read only version implemented and added relevant unit tests